backends/p4tools/modules/testgen/lib/test_backend.h

```diff
@@ -42,6 +42,8 @@ class TestBackEnd {
     /// Test maximum number of tests that are to be produced.
     int64_t maxTests;
 
+    float maxCoverage = 0;
```

Comment:
We have the invariant that every class member/function should have a comment explaining its purpose. Sometimes it is a little redundant but we still try to stick to this principle.